include/rosdiscover-clang/ApiCall/Calls/PublishCall.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include "../RosApiCall.h"
+#include "./Util.h"
+
+namespace rosdiscover {
+namespace api_call {
+
+class PublishCall : public BareRosApiCall {
+public:
+  using BareRosApiCall::BareRosApiCall;
+
+  RosApiCallKind const getKind() const override {
+    return RosApiCallKind::PublishCall;
+  }
+
+  clang::Expr const * getNameExpr() const override {
```

Comment:
We already talked about tackling this problem in a separate PR, but for now, it would be better if this method caused the program to crash (i.e., `abort()`).

Reply:
fixed for both cases

Reply:
Oh, sorry, I should have said: It probably makes sense to write an error to the terminal so that we know that this method is being executed when it shouldn't be. Other than that, we're good to merge

Reply:
fixed